server/src/main/java/com/scalar/db/server/AdminService.java

```diff
@@ -17,7 +17,7 @@
 public class AdminService extends AdminGrpc.AdminImplBase {
   private static final Logger LOGGER = LoggerFactory.getLogger(AdminService.class);
 
-  private static final long MAX_PAUSE_WAIT_TIME_MILLIS = 10000; // 10 seconds
+  private static final long DEFAULT_MAX_PAUSE_WAIT_TIME_MILLIS = 10000; // 10 seconds
```

Comment:
Actually, I was thinking the original max wait time is too short. So can you please increase it to 30 seconds at this time?

Reply:
Of course!
https://github.com/scalar-labs/scalardb/pull/585/commits/f3f1201fa6d284c716b29c7ce20a1192f41b2b1b